README.md

```diff
@@ -1,73 +1,90 @@
 # ethstorage-sdk
-used for uploading and downloading blobs. The sdk is implemented based on [kzg-wasm](https://github.com/ethereumjs/kzg-wasm/), support the browser environment.
+Tool for uploading and downloading data for EthStorage network, utilizing the [EIP-5018](https://eips.ethereum.org/EIPS/eip-5018) standard for data.
```

Comment:
For `ethstorage-sdk`,  I propose it should have two types of interfaces:
 - **Write/Read a Key-Value Pair Using EthStorage Contract with BLOB Transactions**
     - This method is more direct and convenient.
     - However, the data size is limited to the BLOB size.
 - **Support for Unlimited Data Writing/Reading**
     - This method involves deploying a FlatDirectory which can handle unlimited data chunks as files.
     - Users need to deploy the FlatDirectory first.
     - We also need to provide the source code repository for FlatDirectory.